critical-css-generator.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs')
 // Also force inclusion of full width layout as well as two columns - a page may
 // have one but never both
 penthouse({
-    url: 'http://people-clone.local/index.php',
+    url: 'https://demo.local/index.php',
     css: 'style.min.css',
     forceInclude: [
         '.nhsuk-header__inverted',
```

Comment:
Not sure if all the includes here are still valid/used anywhere, first class does not longer exist in `style.min.css` ... NHSUK uses `.nhsuk-header--white` instead of `.nhsuk-header__inverted` - needs checking

Reply:
Commented out .nhsuk-header__inverted. We may need further tickets to investigate.